src/DGmethods/test/Euler/isentropic_vortex_standalone_bc.jl

```diff
@@ -252,24 +268,28 @@ let
     expected_error[2,3] = 1.0298579367557497e-02
     lvls = size(expected_error, 2)
 
-    for DFloat in (Float64,) #Float32)
-      for dim = 2:3
-        err = zeros(DFloat, lvls)
-        for l = 1:lvls
-          Ne = ntuple(j->2^(l-1) * numelem[j], dim)
-          dt = 1e-2 / Ne[1]
-          nsteps = ceil(Int64, timeend / dt)
-          dt = timeend / nsteps
-          err[l] = run(mpicomm, dim, Ne, polynomialorder, timeend, DFloat, dt)
-          @test err[l] ≈ DFloat(expected_error[dim-1, l])
-        end
-        @info begin
-          msg = ""
-          for l = 1:lvls-1
-            rate = log2(err[l]) - log2(err[l+1])
-            msg *= @sprintf("\n  rate for level %d = %e\n", l, rate)
+    for ArrayType in ArrayTypes
+      for DFloat in (Float64,) #Float32)
```

Comment:
Does the `Float32` case work?

Reply:
In theory yes, but we had some problems early on. We plan to revisit this soon.

Reply:
Problems being large background state on a test we no longer have.

Reply:
Ok, sounds good.

Reply:
Features / requests are being tracked in the BL issue #160